### ISSUE TITLE
Provide documentation on how to upgrade interscript gem in interscript-api

### ISSUE THREAD

**Issue by ronaldtse** [interscript/interscript-api#15]
@phuonghuynh can you write in the README how we can do this? Thanks!

**Comment by ronaldtse** [comment]
(and we need to upgrade to interscript v1.6 gem)

**Comment by phuonghuynh** [comment]
Added to [infra-repo](https://github.com/interscript/infrastructure/blob/master/lambda/README.md)

**Comment by phuonghuynh** [comment]
>(and we need to upgrade to interscript v1.6 gem)

Done, you can check it now
```
API_URL="https://api.interscript.com"
curl -i -XPOST ${API_URL} \
  --verbose \
  --data-raw '{info}'
```

**Comment by phuonghuynh** [comment]
Won't fix since we implementing #18
